### PR TITLE
Updated the OSCAL Workshop and Reorganized Side Navigation Bar

### PR DESCRIPTION
**Pull request by mlynnguyen**
1. Added the 4th OSCAL Workshop Markdown file
2. Updated the Mini Workshop file to include all of the 2023 and 2025 workshops
3. Added weightage to the OSCAL Workshops 2, 3, and 4 to reorganize the Side Navigation Bar

### CODE REVIEW

File: src/content/learn/presentations/oscal-workshop-2023-04.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
+---
+title: 4th OSCAL Workshop
+date: 2024-09-3
+heading: 4th Open Security Controls Assessment Language (OSCAL) Workshop
+toc:
+    enabled: true
+weight: 4
+---
+
+{{% usa-tag %}}When{{% /usa-tag %}} May 24th, 2023
```
Comment (selenaxiao-nist) [inline src/content/learn/presentations/oscal-workshop-2023-04.md:10]: Date is inconsistent with parent page
```suggestion
{{% usa-tag %}}When{{% /usa-tag %}} May 23rd, 2023
```

File: src/content/learn/presentations/_index.md [hunk 2]
```diff
@@ -6,13 +6,19 @@ suppresstopiclists: true
...
 - [OSCAL Assessment Models Overview](/presentations/oscal-ap-ar-poam-v3.pdf) presented during the [Lunch with the OSCAL Developers](/contribute/dev-lunch/) - July 2, 2020
-- [NIST OSCAL Workshop](OSCAL-workshop-20191105.pdf) - November 5, 2019
 - [Security Automation Simplified via NIST OSCAL: We're Not in Kansas Anymore](https://www.youtube.com/watch?v=eP8K7piU5UQ) presented at RSA Conference 2018 - April 18, 2018
 - [Automating Security and Compliance via a New Standard of Standards](https://www.youtube.com/watch?v=mo3J0tFxixg) presented at Docker Government Summit 2018 - April 11, 2018
+
+### Blogs
+- [Blog: Innovating Security Compliance Through Open Standards](https://www.easydynamics.com/blog/innovating-security-compliance-through-open-standards/) - July 7, 2021
+- [Blog: The Foundation for Interoperable and Portable Security Automation is Revealed in NIST’s OSCAL Project](https://www.nist.gov/blogs/cybersecurity-insights/foundation-interoperable-and-portable-security-automation-revealed) - May 19. 2021
```
Comment (selenaxiao-nist) [inline src/content/learn/presentations/_index.md:24]: Typo
```suggestion
- [Blog: The Foundation for Interoperable and Portable Security Automation is Revealed in NIST’s OSCAL Project](https://www.nist.gov/blogs/cybersecurity-insights/foundation-interoperable-and-portable-security-automation-revealed) - May 19, 2021
```

**Review by selenaxiao-nist — COMMENTED** [review]
No problems when building site. New page and side navigation work well.
I found 2 small typos, highlighted in my comments to the relevant files.

**Review by iMichaela — APPROVED** [review]
All comments from @selenaxiao-nist were accepted, and the two typos are fixed now.

**Pull request merged**